app/src/routes/v1/chefs.ts

```diff
@@ -9,8 +9,8 @@ const router = express.Router();
 router.use(requireSomeAuth);
 
 // Submission endpoint
-router.get('/submissions', (req: Request, res: Response, next: NextFunction): void => {
-  chefsController.getSubmissions(req, res, next);
+router.get('/export', (req: Request, res: Response, next: NextFunction): void => {
+  chefsController.getFormExport(req, res, next);
```

Comment:
minor point: maybe at some point, consider calling this `submissionExport`.  form export is another chefs feature. i remember from working with chefs it was useful to differentiate between submissions and the form itself.